telegram-bot/main.go

```diff
@@ -252,6 +261,9 @@ func main() {
 				fetchnote(ID, update.Message.Text, *client)
 			case "paste":
 				paste(ID, update.Message)
+			case "me":
+				me(update.Message.From.UserName, ID, update.Message.Text)
```

Comment:
Just pass the whole update.Message then each of it's methods would be available to you.

```suggestion
				me(update.Message, ID)
```